<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1201</width>
     <height>821</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 163, 72, 255), stop:1 rgba(106, 0, 255, 255))}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 72pt &quot;Z003&quot;;color:white;</string>
    </property>
    <property name="text">
     <string>Ytube</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>681</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: oblique 20pt &quot;URW Gothic&quot;;color:white;</string>
    </property>
    <property name="text">
     <string>download youtube video , playlists or channels</string>
    </property>
   </widget>
   <widget class="QPushButton" name="linkBtn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>670</y>
      <width>531</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: italic 18pt &quot;Ubuntu&quot;;
background-color: rgb(98, 160, 234);
color:white;</string>
    </property>
    <property name="text">
     <string>Get link</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>1011</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: italic 18pt &quot;Ubuntu&quot;;
background-color: rgba(0, 0, 0,0.4);
color:white;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Enter your link here</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
